hermes/README.md

```diff
@@ -8,9 +8,14 @@ The app adds `ignite relayer hermes` commands to allow IBC communication between
 
 - Run both chain to be relayed.
 
-- add the hermes relayer app:
+- add the hermes relayer app from remote:
 ```shell
-ignite app install -g ($GOPATH)/src/github.com/ignite/apps/hermes
+ignite app install -g github.com/ignite/apps/hermes
+```
+
+- or clone the repo and add the hermes relayer app local:
+```shell
+ignite app install -g $GOPATH/src/github.com/ignite/apps/hermes
```

Comment:
Maybe something similar to this that uses "install" instead of "add":

```suggestion
- or clone the repository and install the Hermes relayer app from a local directory:
```shell
ignite app install -g PATH/TO/LOCAL/apps/hermes
```